lib/msal-common/src/request/AuthorizationCodeRequest.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * @type AuthorizationCodeRequest: Request object passed by user to acquire a token from the server exchanging a valid authorization code
+ *  (second leg of OAuth2.0 Authorization Code flow)
+ *
+ * scopes:                  A space-separated array of scopes for the same resource.
```

Comment:
Just wanted to say that I appreciate the effort that gets put into the readability of many of these comments, well done